aws/provider.go

```diff
@@ -261,6 +261,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_sqs_queue":                                 dataSourceAwsSqsQueue(),
 			"aws_ssm_document":                              dataSourceAwsSsmDocument(),
 			"aws_ssm_parameter":                             dataSourceAwsSsmParameter(),
+			"aws_ssm_parameters_by_path":                    dataSourceAwsSsmParametersByPath(),
```

Comment:
I could not determine which naming is better `aws_ssm_parameters_by_path` or simply `aws_ssm_parameters`

Reply:
IMO `aws_ssm_parameters` is cleaner but I'd prefer aligning with the aws-cli so `aws_ssm_parameters_by_path` is the wiser choice.

Reply:
differentiating this feature and the single parameter only with one trailing "s" sounds like problematic user/developer experience :)